packages/client/src/forms/user/fieldDefinitions/createUser.ts

```diff
@@ -79,38 +78,38 @@ export function userSectionFormType(): ISerializedFormSection {
         title: userFormMessages.userDetails,
         fields: [
           {
-            name: 'firstNamesEng',
+            name: 'familyNameEng',
```

Comment:
Would it be possible to change these to just `familyName` so dropping the `Eng` and `En` suffixes?

Reply:
Hi @rikukissa this is the [problem](https://github.com/opencrvs/opencrvs-core/blob/a124bb6a67ffa6745dde96429f96d7ef9858a357/packages/client/src/tests/util.tsx#L1445-L1461) that I was having with the test in relation to the request of renaming `familyNameEng` suffixes. In the mocked data example we already had `familyName` property and so trying to rename familyNameEng to `FamilyName` causes conflicts for the object. 

Since we are not using it in the `user-mgnt` database anyways I thought deleting it `familyName: 'হোসেন'` and renaming the other to   `familyName: 'Hossain'` would make sense would you agree?

Reply:
I agree 👍